src/tokenize.c.src

```diff
@@ -235,10 +270,13 @@ tokenizer_core_@type@(tokenizer_state *ts, parser_config *const config)
                 ts->state = TOKENIZE_LINE_END;
             }
             for (; pos < stop; pos++) {
-                if (*pos == '\r' || *pos == '\n') {
+                if (*pos == '\r') {
                     ts->state = TOKENIZE_EAT_CRLF;
                     break;
                 }
+                else if (*pos == '\n') {
+                    ts->state = TOKENIZE_LINE_END;
+                }
```

Comment:
technically an unrelated fix.  A bit hard to test, since we currently pretty much always ignore empty lines...